pcap-int.h

```diff
@@ -511,7 +524,7 @@ void	pcap_oneshot(u_char *, const struct pcap_pkthdr *, const u_char *);
 void	pcap_win32_err_to_str(DWORD, char *);
 #endif
 
-int	install_bpf_program(pcap_t *, struct bpf_program *);
+PCAP_API int	install_bpf_program(pcap_t *, struct bpf_program *);
```

Comment:
@guyharris , because the original WinPcap exported this function. To keep the compatibility, I chose to export it in libpcap for Windows too. See this: https://github.com/wireshark/winpcap/blob/master/wpcap/PRJ/WPCAP.DEF#L70